config/install/core.date_format.fallback.yml

```diff
@@ -0,0 +1,10 @@
+uuid: 0dda3437-336e-4a68-addf-e97fe1732c76
```

Comment:
We have a date format module that we should continue to standardize and store these in. 
https://github.com/SU-SWS/stanford_date_formats/tree/8.x-1.x

Reply:
This is installed as part of `system` https://git.drupalcode.org/project/drupal/tree/8.8.x/core/modules/system/config/install